packages/react/src/themes/teams/components/Dialog/dialogStyles.ts

```diff
@@ -0,0 +1,47 @@
+import { DialogProps } from '../../../../components/Dialog/Dialog'
+import { ComponentStyleFunctionParam, ICSSInJSStyle } from '../../../types'
+import { DialogVariables } from './dialogVariables'
+
+type DialogStyleParams = ComponentStyleFunctionParam<DialogProps, DialogVariables>
+
+export default {
+  root: ({ props: p, variables: v }: DialogStyleParams): ICSSInJSStyle => ({
```

Comment:
As I see we have the same stuff in `base` theme, why we can't reuse it and add only missing things?
I.e. skip `actions`, `overlay` and `content` slots at all...

Reply:
I was originally copied styles to update for redlining for the component. But this PR is just going to be a color/themeing update and the redlining can follow in a separate PR. Will update the styles to not duplicate here.